### PR TITLE
[#33] Avatar storybook

### LINKED ISSUE #33: Avatar
## Tasks
- [ ] Storybook - 2MD

### PR DESCRIPTION
**Pull request by daisy142**
**What this PR does / why we need it**:

- Add storybook and mdx document for Avatar

**Special notes for your reviewer**:

**Which issue(s) this PR fixes**:

Fixes [#33](https://github.com/yorkie-team/yorkie-ui-poc/issues/33)

**Screenshots:**:

Screenshot 2024-01-15 at 15 41 38
Screenshot 2024-01-15 at 15 35 29
Screenshot 2024-01-15 at 15 35 35

**Checklist**:
- [ ] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

File: src/components/ui/avatar/Avatar.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+<Meta of={AvatarStories} />
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Examples](#examples)
+
+## Overview
+
+A graphical representation of the user, often used in profile sections.
```
Comment (vitran12) [inline src/components/ui/avatar/Avatar.mdx]: An avatar is a visual representation of a user, usually in the form of a profile picture, initials, or an icon that represents the user. It serves as a visual identifier for users, making it easier to distinguish between different individuals in a digital environment.

File: src/components/ui/avatar/Avatar.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  }}
+/>
+
+## Props
+
+<Controls />
+
+## Usage
+
+A `Avatar` consists of 3 properties:
```
Comment (vitran12) [inline src/components/ui/avatar/Avatar.mdx]: A `Avatar` object comprises three essential properties:

File: src/components/ui/avatar/Avatar.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+    <Avatar size="sm" src="https://i.pravatar.cc/300" name="John Doe" />
+  `}
+/>
+
+## Examples
+
+### Sizes
+
+Using the `size` prop to change the avatar's size. There are 6 sizes: `xs`, `sm`, `md` (default), `lg`, `xl`, `2xl`.
```
Comment (vitran12) [inline src/components/ui/avatar/Avatar.mdx]: The default value for the `size` prop is `md`. You can use the size prop to modify the avatar's dimensions, with six available options: `xs`, `sm`, `md`, `lg`, `xl`, and `2xl`.

File: src/components/ui/avatar/Avatar.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  <Story of={AvatarStories.AvatarSm} />
+  <Story of={AvatarStories.AvatarMd} />
+  <Story of={AvatarStories.AvatarLg} />
+  <Story of={AvatarStories.AvatarXl} />
+  <Story of={AvatarStories.Avatar2xl} />
+</Canvas>
+
+### Src
+
+Using the `src` prop to import the avatar image.
```
Comment (vitran12) [inline src/components/ui/avatar/Avatar.mdx]: To import the avatar image, you can use the `src` prop.

File: src/components/ui/avatar/Avatar.mdx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  of={AvatarStories.AvatarMd}
+  source={{
+    format: true,
+    code: `<Avatar src="https://i.pravatar.cc/300" name="John Doe"} />`,
+  }}
+/>
+
+### Name
+
+Using the `name` prop to import the avatar name.
```
Comment (vitran12) [inline src/components/ui/avatar/Avatar.mdx]: By utilizing the `name` prop, you can seamlessly import the name for the avatar.

File: src/components/ui/avatar/Avatar.mdx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+<Canvas
+  of={AvatarStories.AvatarMd}
+  source={{
+    format: true,
+    code: `<Avatar src="https://i.pravatar.cc/300" name="John Doe"} />`,
+  }}
+/>
+
+### Basic
+
```
Comment (vitran12) [inline src/components/ui/avatar/Avatar.mdx]: Add description for `Basic`, `With Name Only`, `With No User Data`

**Review by vitran12 — COMMENTED** [review]
Thank you @daisy142 for your work

I left a few comments. Please check them~

**Comment by vitran12** [comment]
You should update description for props 

ASIS:

Screenshot 2024-01-11 at 10 46 30


TOBE ( example) :

Screenshot 2024-01-11 at 10 46 23

**Review by vitran12 — APPROVED** [review]

**Comment by daisy142** [comment]
> Thank you @daisy142 for your work
> 
> I left a few comments. Please check them~

Thanks a lot. I updated. Please help me check it @vitran12

**Review by vitran12 — APPROVED** [review]

**Pull request merged**